packages/cf-component-icon/test/Icon.js

```diff
@@ -40,4 +40,12 @@ describe('Icon', function() {
       <i className="cf-icon cf-icon--info-sign"></i>
     );
   });
+
+    it('should render role', function() {
+    assertEqualJSX(
```

Comment:
lol I'll fix.  Had to edit in sublime cause intellij kept crashing for some reason.  Need you to add me as a publisher to all the packages though.